web/modules/custom/mymodule/src/Form/MymoduleSettingsForm.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+namespace Drupal\mymodule\Form;
+
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Mymodule settings form class is for form with configuration.
+ */
+class MymoduleSettingsForm extends ConfigFormBase {
+
+  /**
+   * Method to call the module settings.
+   *
+   * @return array
+   *   It will return the array of settings.
+   */
+  protected function getEditableConfigNames() {
+    return [
+      'mymodule.settings',
+    ];
+  }
+
+  /**
+   * Method to call the formid.
+   *
+   * @return string
+   *   It will return the form id.
+   */
+  public function getFormId() {
```

Comment:
@anitha-innoraft - This is a method whose documentation is already present in `ConfigFormBase` class. We don't need to re-write the documentation. You can use https://www.drupal.org/docs/develop/standards/api-documentation-and-comment-standards#inheritdoc
```
/**
 * {@inheritdoc}
 */
```